spec/reset_password_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+describe 'password reset', type: :feature, js: true do
+  it 'resets password at LOA1' do
+    email_address = create_new_account
+
+    visit idp_logout_url
+
+    visit idp_reset_password_url
+
+    expect(page).to have_content('Forgot')
```

Comment:
Or, if we want to have this expectation, we could make 2 separate test? I know @monfresh likes to consolidate tests for speed, but I find specs with expectations in multiple places to be less readable and therefore less helpful